tests/unit-tests/dds/security/AccessControlTest.cpp

```diff
@@ -26,8 +26,8 @@ using namespace testing;
 static const char* Expected_Permissions_Token_Class_Id ="DDS:Access:Permissions:1.0";
```

Comment:
Name should be `AccessControlBuiltInImpl.cpp` to match the code it's testing. The lint script will have to match the files using the names.